components/family-studies/api/src/main/java/org/phenotips/studies/family/script/FamilyScriptService.java

```diff
@@ -53,12 +53,34 @@
      *
      * @return Family object corresponding to the newly created family.
      */
-    public Family createFamily()
+    public Family create()
     {
         return this.familyTools.createFamily();
     }
 
     /**
+     * Creates an empty family.
+     *
+     * @return Family object corresponding to the newly created family.
+     */
```

Comment:
Deprecate?

Reply:
Yes, but in next milestone, to avoid warnings in the logs.